decentraland/unity-renderer
# Top notification initial changes

## What does this PR change?

Fix #2933 

...

## How to test the changes?

1. Go to: https://play.decentraland.org/?NETWORK=mainnet&realm=dg&position=100%2C101&ENABLE_NOTIFICATION_CHAT=&kernel-branch=feat%2Fadd-friends-lazy-load&renderer-branch=feat/top-notification-rework
2. Open the chat
3. Generate notifications by pressing k and g
4. Verify that the behaviour described in the issue #2933 works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
